torch/__init__.py

```diff
@@ -300,10 +300,69 @@ def set_default_dtype(d):
     _C._set_default_dtype(d)
 
 def set_deterministic(d):
-    r"""Sets a global flag to force all operations to use a deterministic
-    implementation if available. If an operation that does not have a
-    deterministic implementation is called while this setting is True, the
-    operation will throw a RuntimeError.
+    r"""Sets a global flag to force normally nondeterministic operations to
+    use a deterministic implementation, if available. Or if any of these
+    operations do not have a deterministic implementation, they will throw
+    a :class:`RuntimeError`.
+
+    .. warning::
+        This feature is a work in progress, so it does not affect every
+        nondeterministic operation yet. All affected operations are listed below.
+        Any other operations in PyTorch that call the listed operations
+        internally are also affected.
+
+    The following normally-nondeterministic operations will act
+    deterministically when `d=True`:
+
+        * Convolution operations
+        * :func:`torch.bmm` for sparse-dense CUDA inputs
```

Comment:
batched matrix multiply is a convolution operation?

Reply:
That's not what this is supposed to mean, they are separate bullet points.

Reply:
Oh I see. I think I was confused because "Convolution operations" sounds like a heading while the other entries are the actual functions. Maybe an "(e.g. ...)" would help?

Reply:
Maybe the change I just pushed makes it more clear?

Reply:
Or perhaps I should just actually list all of the convolution operations that it affects. I'd have to learn what they are

Reply:
I replaced "Convolution operations" with a full list of the operations.